src/io/github/perrymant/moneymaker/BudgetTest.java

```diff
@@ -0,0 +1,130 @@
+package io.github.perrymant.moneymaker;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class BudgetTest {
+
+    private Balance balance = new Balance();
+    private List<Transaction> transactions = new ArrayList<>();
+
+    @Test
+    public void canProjectBalance() {
+        makeBudget();
+    }
+
+    @Test
+    public void startBalanceIsZero() {
+        balance = new Balance();
+        assertEquals(0, balance.get());
+        makeBudget();
+    }
+
+    @Test
+    public void startCanChangeBalanceValue_Credit() {
+        transactions.add(makeCreditTransaction());
+        transactions.add(makeCreditTransaction());
+        Budget target = makeBudget();
+        assertEquals(0, balance.get());
+        target.updateBalance();
+        assertEquals(300, balance.get());
+    }
+
+    @Test
+    public void createDebitTransaction() {
+        transactions.add(makeDebitTransaction());
+        Budget target = makeBudget();
+        assertEquals(0, balance.get());
+        target.updateBalance();
+        assertEquals(-20, balance.get());
+    }
+
+    // GivenATransaction_ReturnReportStringFormat
+
+    @Test
+    public void checkTimeFormatting() {
+        Transaction transaction = new Transaction();
+        assertEquals("2018-08-06", transaction.getTimeNow());
+    }
+
+    @Test
+    public void givenTransaction_GetDescription() {
```

Comment:
This doesn't test `Budget` so it sounds to me like it doesn't belong here.
Applies to multiple tests in this file.